src/badge/README.md

```diff
@@ -0,0 +1,43 @@
+# Badge
+
+[View prototype on CodePen][codepen].
+
+[codepen]: https://codepen.io/thoughtbot/pen/ZEzROvy
+
+## Usage
+
+### Default
+
+```html
+<span class="tbds-badge">Badge Label</span>
+```
+
+### Pill Badge
```

Comment:
Something I’d like to start chipping away at is moving this project from a component library to more of a design system. So I’m curious: do you have thoughts on when a “pill” badge should be used instead of the “default”?

This isn’t a blocker for merging…

Reply:
I was just thinking variety here, I know some people have aesthetic preferences in either direction. Although thinking about it now, should we have badges have more rounded corners across the board? If and when we do tags that will help distinguish their differences visually.

Reply:
> I was just thinking variety here, I know some people have aesthetic preferences in either direction.

I think the system should provide intent with its design and other offerings, instead of variety. Meaning, the components we provide are tied to a specific problem space, which is documented and consumers of the system can understand why and when a component should be used.

> Although thinking about it now, should we have badges have more rounded corners across the board? 

Perhaps! It can be helpful to work backward from the desired intent. If Constable is our current situation in which Badges are needed, let’s use that real application and design for/from it. So the question becomes: does Constable need rounded or non-rounded Badges?

> If and when we do tags that will help distinguish their differences visually.

Yes, and I would encourage us to document the intention and purpose behind each of the components.

Reply:
Cool. I added some explainer documentation to help communicate usage, including some language about how badges aren't tags.